maintainers.md

```diff
@@ -0,0 +1,9 @@
+# Maintainers of OpenIntegrationEngine project
+
+- [Jonathan Bartels](https://github.com/jonbartels)
+- [Kaur Palang](https://github.com/kpalang)
+- [Kaur Palang](https://github.com/kpalang)
+- [Kaur Palang](https://github.com/kpalang)
+
+
```

Comment:
Think you had a copy/paste error here. List Kaur once. Please add me. Remove extra trailing new lines.